docs/reference/search/request/stored-fields.asciidoc

```diff
@@ -40,7 +40,7 @@ GET /_search
 
 If the requested fields are not stored (`store` mapping set to `false`), they will be ignored.
 
-Field values fetched from the document it self are always returned as an array. Metadata fields like `_routing` and
+Field values fetched from the document itself are always returned as an array. Whereas metadata fields like `_routing` and
```

Comment:
I'd change "Field values" to "Stored field values"

"Whereas" doesn't really work here - it should be ", whereas"  in this context.  That said, I prefer the sentence without whereas.  The intention is to make the contrast between the two sentences easy to spot.